### PR TITLE
Code examples for episode 005

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: ep005/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# [the python @property decorator (beginner - intermediate)](https://youtu.be/orp6bhe4i00)
+
+In this video I explain how python implements getters and setters as well as an example where you might use them!  As a bonus I also show @cached_property
```
Comment (asottile) [inline ep005/README.md:3]: probably want to put this in backticks so it doesn't @-mention github.com/cached_property

**Review by asottile — APPROVED** [review]

File: ep005/README.md (hunk 1 above)
Comment (int3l) [reply ep005/README.md:3]: It looks like it doesn't do it, but ok: https://github.com/int3l/explains/blob/e4a95a45f3bacb3a01d7e777f5982c457a45258f/ep005/README.md

Comment (int3l) [reply ep005/README.md]: Maybe if someone create account with this nickname, it will start working 👍

**Pull request merged**